sdks/python/apache_beam/transforms/__init__.py

```diff
@@ -23,3 +23,9 @@
 from apache_beam.transforms.ptransform import *
 from apache_beam.transforms.timeutil import TimeDomain
 from apache_beam.transforms.util import *
+
+# No backwards compatibility guarantees.
+try:
+  from apache_beam.transforms.cy_dataflow_distribution_counter import DataflowDistributionCounter
+except ImportError:
+  from apache_beam.transforms.cy_combiners import DataflowDistributionCounter
```

Comment:
<!--new_thread; commit:70a22e2b9aaf49c4f9fc6cc86ed5bf10093823e7; resolved:0-->
It's counter-intuitive that this is the one that's slow. Also, do we need this in the top -level apache_beam.transform?

Reply:
yes, this will first trying to import cython version, if failed, then import python version. Then we can avoid a bunch of try-except clauses when importing to other modules